src/Exec/ExecTrait.php

```diff
@@ -88,7 +89,11 @@ public function startBrowser($uri = null, $sleep = 0, $port = false, $browser =
      */
     public static function programExists($program)
     {
-        $process = Drush::shell("command -v $program");
+        $cmd = "command -v $program";
+        if (Escape::isWindows()) {
+            $cmd = "where $program";
```

Comment:
I think I was in error recommending that we use `where` here. If `command` works, then it would be preferable to remove this.

Reply:
`command -v $program` doesn't work for mysql on windows
`mysql -V` works

What do you prefer?

Reply:
OK, I guess I misunderstood the comment in the other issue. If `command -v` does not work, then I prefer to leave this as-is, and continue to use `where`